framer/SVGGroup.coffee

```diff
@@ -1,59 +1,59 @@
 {Layer, layerProperty} = require "./Layer"
 {Color} = require "./Color"
 {SVG} = require "./SVG"
+
+{SVGBaseLayer} = require "./SVGBaseLayer"
 {SVGPath} = require "./SVGPath"
 
-class SVGGroup extends Layer
+areEqual = (a, b) ->
```

Comment:
This is an oddly specific equality check. What’s not in lodash / already in Framer that requires this?

Reply:
Fixed by creating `Utils.equal`